images/api-server/Dockerfile

```diff
@@ -14,7 +14,8 @@ RUN wget -q https://github.com/kamu-data/kamu-platform/releases/download/v${API_
 
 
 ENTRYPOINT ["/usr/bin/tini", "--"]
-# TODO: replace on S3 version
-#    CMD ["/opt/kamu/kamu-api-server --repo-url s3://<bucket-path> run --address 0.0.0.0 --http-port 80"]
-CMD ["/opt/kamu/kamu-api-server --local-repo /opt/kamu/workspace run --address 0.0.0.0 --http-port 80"]
+# S3-based:
+CMD ["/opt/kamu/kamu-api-server --repo-url s3+https://s3.us-west-2.amazonaws.com/datasets.kamu.dev/ run --address 0.0.0.0 --http-port 80"]
```

Comment:
Will this break Dmytro's workflow of testing against `latsest-with-data` image?

I'd prefer the image to either use local repo, or to demand an explicit repo URL parameter, rather than hard-coding S3 our own URLs.

We should also add `repoUrl` parameter to the helm chart of API server and override the container command.

Reply:
This will not break Dmytro's testing flow, since he is launching the image with data locally.